enforcer-rules/pom.xml

```diff
@@ -112,22 +112,6 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
you cannot remove that unfortunately without breaking backwards compatibility (maybe add a TODO already that this should be removed with m-enforcer 4.0.0)

Reply:
Ok, will restore; please keep the review comments coming, I'll process them later.